### PR TITLE
Using CSV to import words its means

### PR DESCRIPTION
**Pull request by nhs3108**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,8 +1,14 @@
 Rails.application.routes.draw do
+
+  namespace :admin do
+  get 'imports/create'
```
Comment (dieunb) [inline config/routes.rb]: indent

File: app/controllers/admin/imports_controller.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  private
+  def import_csv file
+    CSV.open(file.path, "rt", headers: true).each do |row|
+      category_name = row["category_name"]
+      category_description = row["category_description"]
+      word_word = row["word"]
+      word_mean = row["mean"]
+      is_correct = row["is_correct"]
+
+      category = save_changes "Category".safe_constantize, {name: category_name}, {name: category_name, description: category_description}
```
Comment (ThuBM) [inline app/controllers/admin/imports_controller.rb]: chia lam 2 dong

File: app/controllers/admin/imports_controller.rb [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+      word = save_changes "Word".safe_constantize, {word: word_word}, {word: word_word, category_id: category.id}
```
Comment (ThuBM) [inline app/controllers/admin/imports_controller.rb]: chia lam 2 dong

File: app/controllers/admin/imports_controller.rb [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+      save_changes "Answer".safe_constantize, {mean: word_mean, word_id: word.id}, {word_id: word.id, mean: word_mean, is_correct: is_correct}
```
Comment (ThuBM) [inline app/controllers/admin/imports_controller.rb]: Chia lam 2 dong

File: app/controllers/admin/imports_controller.rb [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+  private
+  def import_csv file
+    CSV.open(file.path, "rt", headers: true).each do |row|
+      category_name = row["category_name"]
+      category_description = row["category_description"]
+      word_word = row["word"]
+      word_mean = row["mean"]
+      is_correct = row["is_correct"]
+
+      category = save_changes Category, {name: category_name}, {name: category_name, description: category_description}
```
Comment (ThuBM) [inline app/controllers/admin/imports_controller.rb]: split to 2 rows. Max length of 1 rows is 80 characters.

File: app/controllers/admin/imports_controller.rb [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+      word = save_changes Word, {word: word_word}, {word: word_word, category_id: category.id}
+      save_changes Answer, {mean: word_mean, word_id: word.id}, {word_id: word.id, mean: word_mean, is_correct: is_correct}
```
Comment (ThuBM) [inline app/controllers/admin/imports_controller.rb]: split to 2 rows

File: app/controllers/admin/imports_controller.rb [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+    end
+  end
+
+  def save_changes model, fields_to_check, data
+    if(existedCategory = model.find_by fields_to_check)
```
Comment (ThuBM) [inline app/controllers/admin/imports_controller.rb]: add space after if

**Pull request merged**